website/app/modifiers/doc-track-event.js

```diff
@@ -9,14 +9,10 @@ import { inject as service } from '@ember/service';
 import { assert } from '@ember/debug';
 
 export default class DocTrackEvent extends Modifier {
-  @service fastboot;
+  @service eventTracking;
 
   modify(element, _positional, named) {
-    if (
-      // Only attempt to do something if we are in the right environment
-      this.fastboot.isFastBoot ||
-      window.fathom == null
-    ) {
+    if (!this.eventTracking.isEnabled) {
       // comment this line if you want to test in your local environment
```

Comment:
[nit] I think this was added via suggestion in a previous PR, but trying this out I don't see the events actually working.

Reply:
🤔

Reply:
Are you adding something to actually log the string or how are you seeing the event?

Reply:
AFAIK you would need to comment it out as well as adding a log.

Reply:
Yes, as @zamoore says, I'm commenting out the call to Fathom and using a simple `console.log` instead.

Reply:
Ok. I think I was reading this too literally then as the _only_ thing you needed to do.